scripts/veloxbench/veloxbench/cpp_micro_benchmarks.py

```diff
@@ -0,0 +1,121 @@
+import copy
+import json
+import os
+import tempfile
+
+import conbench.runner
+
+from veloxbench import _benchmark
+
+RUN_OPTIONS = {
+    "iterations": {
+        "default": None,
+        "type": int,
+        "help": "Number of iterations of each benchmark.",
+    },
+}
+
+
+# TODO: what other COMMON_OPTIONS?
+# https://github.com/ursacomputing/benchmarks/blob/033eee0951adbf41931a2de95caccbac887da6ff/benchmarks/cpp_micro_benchmarks.py#L18-L59
+# TODO: the suite filter won't quite work yet either
+COMMON_OPTIONS = {
+    "suite-filter": {
+        "default": None,
+        "type": str,
+        "help": "Regex filtering benchmark suites.",
+    },
+}
+
+
+def get_run_command(filename, options):
+    # TODO: we'll probably want to unify these runner scripts
```

Comment:
Unifying here I mean both this code and that in `scripts/benchmark-runner.py`. If we wanted we could pull the runner script into this code and deduplicate | reduce the abstractions. Or we can keep them separate if folks like using that runner script on its own now.

Reply:
I don't think anyone is using that script for now (other than the circleCI job). Should be fine for us to merge them.